irr/src/CMakeLists.txt

```diff
@@ -337,6 +340,8 @@ add_library(IRROBJ OBJECT
 	CMeshCache.cpp
 )
 
+target_link_libraries(IRROBJ PRIVATE IRRMESHOBJ)
```

Comment:
huh?

Reply:
It needs the transitive include for the tiniergltf header.

Reply:
I think I have resolved this, but please check whether this is satisfactory.

Reply:
this works. The other alternative would be to have `CGLTFMeshFileLoader.h` strictly only contain the interface.

Reply:
When we switched from tinygltf to tiniergltf I think we needed some tiniegltf types in the header, but if we could avoid that it would be preferable. @appgurueu